### ISSUE TITLE
Problem making index

### ISSUE THREAD

**Issue by CormacKinsella** [bioinformatics-centre/kaiju#61]
Hi there, 
I'm on MacOS, I used makeDB.sh to download the progenomes and viral protein set, but after this I get the following error message: 
(command used was: makeDB.sh -p -v -t 1)

Extracting file taxdump.tar.gz
Creating directory genomes/
Extracting protein sequences from downloaded files...
xargs: illegal option -- i
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]

What am I doing wrong?

Thanks in advance!

**Comment by pmenzel** [comment]
Hi,
that is a problem in makeDB.sh when running it on MacOS.
Will try to fix it quickly!
Peter

**Comment by charlesreid1** [comment]
I'm on the case...

**Comment by charlesreid1** [comment]
Hi @pmenzel, your proposed fix worked. Changing the xargs commands in lines 238 and 256 to

```
xargs -n 1 -P $parallelConversions -IXX gbk2faa.pl XX XX.faa
```

ran through the whole makeDB process smoothly.

**Comment by charlesreid1** [comment]
Also, just for informational purposes, if you're bewildered by the way that Mac command line utilities only share 90% of the command line flags as their Linux counterparts (xargs, grep, ls, sed, etc), it's because Mac is using the BSD version of all those utilities (b/c underlying Darwin), while Linux uses the GNU version. A quick check online for the man page for the BSD versions should do it.

**Comment by pmenzel** [comment]
Thanks @charlesreid1 for testing and the PR!

@CormacKinsella Please get the latest version from github via
```
git clone https://github.com/bioinformatics-centre/kaiju.git
```
and then compile it with
```
cd kaiju/src
make
```

Afterwards, try running `kaiju/bin/makeDB.sh -p -v` again.

**Comment by CormacKinsella** [comment]
many thanks both!

**Comment by pmenzel** [comment]
Fix is  now included in version 1.6.2.
